src/main.js

```diff
@@ -51,43 +52,57 @@ bot.on('message', (message) => {
     switch(args[0]){ 
       // Faucet commands
       case 'goerliEth': {
+        let embed = new Discord.MessageEmbed().setColor(3447003).setTimestamp();
         if (args[1] == null) {
-          message.channel.send('Goerli ETH adress required.')
+          embed.setDescription('Goerli ETH address is required.');
+          message.lineReply(embed);
+//           message.channel.send({embed});
           break;
         } else if (!web3.utils.isAddress(args[1])) {
-          message.channel.send('Address is not the proper format.  Please double check.')
+          embed.setDescription('Address is not in the proper format. Please double check.');
+          message.lineReply(embed);
+//           message.channel.send({embed});
           break;
         } else {
           // 3rd arg is amount of eth, 4th is whether to run custom checks
-          bot.commands.get('goerliBot').execute(message, args, 1, false);
+          bot.commands.get('goerliBot').execute(message, args, 0.001, true);
         }
         break;
       }
 
       // Other commands
       case 'help': {
         console.log("help called");
-        message.channel.send(EMBEDDED_HELP_MESSAGE);
+        message.lineReply(EMBEDDED_HELP_MESSAGE);
+//         message.channel.send(EMBEDDED_HELP_MESSAGE);
         break;
       }
       
       case 'mod': {
         // Tag the moderators
         console.log("mod called");
         // Uncomment below and add discord ids if you'd like to be tagged
-        // message.channel.send('Alerting the maintainers - <@discord-id> and <@discord-id> come check this out.');
+        
+        // let embed = new Discord.MessageEmbed()
+        //     .setDescription('Alerting the maintainers - @!<discord-id> and @!<@discord-id> come check this out.')
+        //     .setColor(3447003).setTimestamp();
+        // message.channel.send({embed});
         break;
       }
 
       // For fun :)
       case 'dance': {
         console.log("dance called");
-        message.channel.send('https://c.tenor.com/fJh-W38iA3oAAAAM/dance-kid.gif');
+        let embed = new Discord.MessageEmbed().setImage('https://c.tenor.com/fJh-W38iA3oAAAAM/dance-kid.gif').setColor(3447003).setTimestamp();
```

Comment:
Is it possible to use base64 format of image instead of requesting it from external source?